samples/rabbitmq/simple/sample.md

```diff
@@ -10,21 +10,18 @@ related:
 ---
 
 
-## Code walk-through
+## Prerequisites
 
-This sample shows a basic usage of RabbitMQ as a transport for NServiceBus.
+Ensure you have a running instance of RabbitMQ.
 
 
-### Enable RabbitMQ
+## Code walk-through
 
-snippet:ConfigureRabbit
+This sample shows basic usage of RabbitMQ as a transport for NServiceBus. The application sends an empty message to itself, via the RabbitMQ broker, and writes to the console when the message is received.
 
 
-### Configuring RabbitMQ
+### Configuration
```

Comment:
Should this be another section on its own (##) instead of a sub-heading under "Code walk-through" (###) ?